flow-typed/types/kaltura-player-options.js

```diff
@@ -34,7 +34,8 @@ declare type PartialKPOptionsObject = {
   playlist?: KPPlaylistObject,
   dimensions?: PKDimensionsConfig,
   ui?: KPUIOptionsObject,
-  cast?: {[key: string]: any}
+  cast?: {[key: string]: any},
+  productVersion?: string
```

Comment:
this value shouldn't be configurable, please remove from config object

Reply:
on second thought, we pass it in the config object? we need to extract it to be internal cause it can be easily changed as a config.

Reply:
do you mean to delete it from the config?